scripts/build.sh

```diff
@@ -20,6 +20,8 @@ ansible_image_version=${CONTIV_ANSIBLE_IMAGE_VERSION:-$contiv_version}
 auth_proxy_version=${CONTIV_API_PROXY_VERSION:-$contiv_version}
 docker_version=${CONTIV_DOCKER_VERSION:-1.12.6}
 etcd_version=${CONTIV_ETCD_VERSION:-v2.3.8}
+contiv_ansible_commit=${CONTIV_ANSIBLE_COMMIT:-3cb33ce9629d5b0d27c630581bc3a26e08a0d119}
```

Comment:
Can you make this default to HEAD, please?

Reply:
the specific SHA is purposeful, in particular, I want to push a change to both install and ansible that changes the contract slightly, so this commit (and maybe others depending on timing) need to point to the current HEAD, then we can merge a new ansible without breaking install, then  merge changes to install.

We will need to update this in install as we make changes to ansible to see the updates, but that's a good thing.

Reply:
This would make it a whole lot more complicated when testing. This should be set only for releases. We can refuse to make a release if this is set to HEAD.

Reply:
This should default to HEAD on the master branch.  As part of our release process, we edit this file to adjust `contiv_version` and `v2plugin_version`, and we can also update this variable at that time.

Reply:
nvm, just saw discussion on Slack 😄